src/frontend/apps/desk/src/i18n/utils.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  BASE_LANGUAGE,
+  LANGUAGES_ALLOWED,
+  LANGUAGE_LOCAL_STORAGE,
+} from './conf';
+
+export const splitLocaleCode = (language: string) => {
+  const locale = language.split(/[-_]/);
+  return {
+    language: locale[0],
+    region: locale[1],
```

Comment:
This key can be undefined right ?

Reply:
https://github.com/numerique-gouv/people/pull/17/commits/df3b394f36cc341a13b28f8d12520902dc7df5f2#diff-4199f89f406d85e6fd929a6d2de6737f8b041b22bbc9903e1f0764287354c8a5R11

Reply:
Like that (`const [language, locale] = lng.split(/[-_]/);`) locale has the type `string`, when actually locale can be `string | undefined`.